src/ucp/core/ucp_context.c

```diff
@@ -19,6 +20,10 @@
 #include <string.h>
 
 
+extern const ucp_proto_t ucp_tag_eager_sync_proto;
+extern const ucp_proto_t ucp_tag_eager_sync_unsupport_proto;
+extern const ucp_proto_t *ucp_tag_eager_sync_proto_p;
```

Comment:
I'd add this pointer to the context instead. In theory we may have several contexts and not all of them may need FAULT_TOLERANCE feature